src/resources/ResourceQuotas/ResourceQuotaDetails.tsx

```diff
@@ -41,51 +48,94 @@ export type ResourceQuotaProps = {
 
 export default function ResourceQuotaDetails(props: any) {
   const { t } = useTranslation();
+  const { podsMetrics } = usePodsMetricsQuery(props.namespace);
 
   const customComponents = [
-    (resource: ResourceQuotaProps) => {
-      return (
-        <React.Fragment key="resource-quota-details">
-          {(resource.spec.scopes || resource.spec.scopeSelector) && (
-            <UI5Panel title={t('common.headers.specification')}>
-              {resource.spec?.scopes && (
-                <LayoutPanelRow
-                  name={t('resource-quotas.headers.scopes')}
-                  value={<Tokens tokens={resource.spec.scopes} />}
-                />
-              )}
-              {resource.spec?.scopeSelector && (
-                <UI5Panel title={t('resource-quotas.headers.scope-selectors')}>
-                  {resource.spec.scopeSelector?.matchExpressions?.map(scope => (
-                    <>
-                      <Title
-                        level="H6"
-                        size="H6"
-                        className="resource-quota-spec-subheader sap-margin-small"
-                      >
-                        {scope.scopeName}
-                      </Title>
+    (resource: ResourceQuotaProps) => (
```

Comment:
Would you like to rename `ResourceQuotaProps` to `ResourceQuota` because this type looks like k8s `ResourceQuota`